.github/workflows/lint.yml

```diff
@@ -9,6 +9,16 @@ on:
   pull_request:
 
 jobs:
+  codespell:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This is very appreciated, thank you very much. I didn't expect to see so many grammar errors... _alot!_ :smile: 

The github action doesn't describe how to run the tool locally. Would it be possible to implement it in `tox.ini` instead, to make the command available to both command line (and pre-commit scripts, for who use them) and use it in the pipeline, by a step:

```yaml
      - name: Check grammar
        run: tox -e codespell
```

like the ones below in this file?